cli/cmd/options.go

```diff
@@ -100,7 +100,8 @@ func getDefaultEndpoint(ip, port string) string {
 	if ip == "" {
 		var err error
 		if ip, err = utils.GetIPAddress(); err != nil {
-			panic(err)
+			//fmt.Fprintf(os.Stderr, "ERROR: Unable to get default endpoint: %s\n", err)
+			return ""
```

Comment:
I did not use Fatal here so simple commands like 'serviced version' and 'serviced help' would run.  This "" gets placed in options.Endpoint, then when the server actually tries to start up and discovers it doesn't have the info it needs, it Fatals out.  (Even printing the error message that I commented out is very distracting and annoying for those simple commands.)